lib/chef-cli/cookbook_profiler/git.rb

```diff
@@ -103,6 +104,17 @@ def current_branch
       end
 
       private
+      
+      def repo_directory
+        @repo_directory ||=
```

Comment:
Indentation is weird, the following more compact form seems clearer to me and fix the various other comments

```ruby
    def repo_directory
      @repo_directory ||= Pathname.new(@cookbook_path).ascend do |parent_dir|
        possible_git_dir = File.join(parent_dir, '.git')
        break possible_git_dir if File.directory?(possible_git_dir )
      end
    end
```

Reply:
Done, thx for the suggestion !